syncd/scripts/saidump.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+function debug()
```

Comment:
Not sure if this is necessary, maybe you can use direct logger command instead of this wrapper, no need for explicit path

Reply:
Fix it.